src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
```

Comment:
The class name 'Rectangle' is appropriate as it is a noun, but make sure that any interfaces it implements have names that do not use verbs. For example, if there is an interface for calculating area, it should not be named 'CalculateArea' but rather something like 'AreaCalculable' or 'Shape' if it includes other shape-related behaviors.